src/core.rs

```diff
@@ -32,6 +32,52 @@ pub(crate) mod marker {
     pub const BREAK: u8   = 0xff;
 }
 
+
+#[cfg(feature = "use_alloc")]
+pub mod buf_writer {
```

Comment:
I thought there could be a `core::utils` mod that could be put together with `SliceReader`.

Reply:
Done.